modules/mono/editor/script_templates/EditorScenePostImport/basic_import_script.cs

```diff
@@ -0,0 +1,18 @@
+// meta-description: Basic import script template
+
+#if TOOLS
+using _BINDINGS_NAMESPACE_;
+using System;
+
+[Tool]
+public partial class _CLASS_ : _BASE_
+{
+    public override Object _PostImport(Node scene)
+    {
+        // Modify the contents of the scene upon import. For example, setting up LODs:
+//        scene.GetNode<MeshInstance3D>("HighPolyMesh").DrawDistanceEnd = 5.0
+//        scene.GetNode<MeshInstance3D>("LowPolyMesh").DrawDistanceBegin = 5.0
```

Comment:
I think these should stay aligned despite the comment characters - IDEs are then typically clever enough to uncomment the line while keeping the text indented.
```suggestion
//      scene.GetNode<MeshInstance3D>("HighPolyMesh").DrawDistanceEnd = 5.0
//      scene.GetNode<MeshInstance3D>("LowPolyMesh").DrawDistanceBegin = 5.0
```

Reply:
Got it.